exonum-java-binding/benchmarks/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2014, Oracle America, Inc.
```

Comment:
Why Oracle copyright?

Reply:
Because the archetype, with which the project is generated, includes this header into the pom. IANAL, but it does not seem to restrict the license we can apply to the module. Also, we do not distribute this module in binary.